assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -458,6 +458,15 @@ che.infra.kubernetes.pvc.jobs.memorylimit=250Mi
 # https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/storage.html#pv-access-modes
 che.infra.kubernetes.pvc.access_mode=ReadWriteOnce
 
+# Defines whether Che Server should wait workspaces PVCs to become bound after creating.
+# It's used by all PVC strategies.
+#
+# It should be set to `false` in case if `volumeBindingMode` is configured to `WaitForFirstConsumer`
+# otherwise workspace starts will hangs up on phase of waiting PVCs.
+#
+# Default value is false(means that PVCs should not be waited to be bound)
+che.infra.kubernetes.pvc.wait_bound=true
```

Comment:
According to comment above it should be false, shouldn't it?

Reply:
See https://github.com/eclipse/che/issues/12889#issuecomment-496558203

Reply:
Nope... Acording to https://github.com/eclipse/che/issues/12889#issuecomment-496388486 and followup Mario's comment, it should be true ;-)

Reply:
Dmytro's comment was about doc for a property but about the value. Thanks for catching it. Fixed